### PR TITLE
chore: Add resources requests and limits

### PR DESCRIPTION
**Pull request by kahirokunn**
Fixes #

## Proposed Changes

* If resources requests are left out, they will be given a lower priority when scheduled, so we have assigned them to all.
*
*

**Release Note**

```release-note
chore: Add resources requests and limits
```

### CODE REVIEW

File: config/charts/knative-operator/templates/operator.yaml [hunk 1]
```diff
@@ -6245,5 +6245,12 @@ spec:
           ports:
             - name: metrics
               containerPort: 9090
+          resources:
```
Comment (matzew) [inline config/charts/knative-operator/templates/operator.yaml:6248]: Do these numbers come from any verification or such?

Comment (kahirokunn) [reply config/charts/knative-operator/templates/operator.yaml:6248]: Yes, I have verified this in my environment on a metrics basis and it works well with this value.

Comment (kahirokunn) [reply config/charts/knative-operator/templates/operator.yaml:6248]: If it's not a problem, I'll try to fix the unit test.

Comment (kahirokunn) [reply config/charts/knative-operator/templates/operator.yaml:6248]: @matzew Your opinion.
Does it seem safe to proceed with implementation?
Thx.

**Comment by houshengbo** [comment]
@kahirokunn we ultimately added `requests and limits`, as in https://github.com/knative/operator/blob/main/config/manager/operator.yaml#L41

**Comment by kahirokunn** [comment]
Thanks!

**Pull request closed without merging**